packages/editor/src/components/document-outline/test/index.js

```diff
@@ -78,7 +78,11 @@ describe( 'DocumentOutline', () => {
 		} );
 
 		it( 'should not render when no heading blocks provided', () => {
-			const blocks = [ paragraph ];
+			const blocks = [ paragraph ].map( ( block, index ) => {
+				// Change client IDs to a predictable value
+				block.clientId = '_clientId_' + index;
+				return block;
```

Comment:
As written, the `Array#map` is redundant vs. just `forEach` since you're mutating the object anyways.

As a map, I'd expect: `return { ...block, clientId: '_clientId_' + index };`

Reply:
i'll give that a try, thanks

Reply:
addressed in https://github.com/WordPress/gutenberg/pull/10815/commits/b64b2ed2c42a1e7635f8ce5da5649b6feb99555e